src/System.CommandLine.Subsystems/Subsystems/Annotations/AnnotationAccessor.cs

```diff
@@ -8,12 +8,33 @@ namespace System.CommandLine.Subsystems.Annotations;
 /// <summary>
 /// Allows associating an annotation with a <see cref="CliSymbol"/>. The annotation will be stored by the accessor's owner <see cref="CliSubsystem"/>.
 /// </summary>
+/// <remarks>
+/// The annotation will be stored by the accessor's owner <see cref="CliSubsystem"/>. 
+/// </summary>
+/// <typeparam name="TValue">The type of value to be stored</typeparam>
+/// <param name="owner">The subsystem that this annotation store data for.</param>
+/// <param name="id">The identifier for this annotation, since subsystems may have multiple annotations.</param>
 public struct AnnotationAccessor<TValue>(CliSubsystem owner, AnnotationId<TValue> id)
```

Comment:
Just as a heads up, this will have conflicts with https://github.com/dotnet/command-line-api/pull/2412

Reply:
Did you also add a XML Comment? I can remove mine if you did.

Reply:
I did not, but am happy to bring these comments over to my pr if you want to drop them to make it easier since I should add some for the new parameter I add and the new `Get` method anyways

Reply:
Let's do it that way. Copy away and let me know when I should delete them.

Reply:
Copying complete :)

https://github.com/dotnet/command-line-api/pull/2412/commits/ff932a49e1b7d5fa03e40d645bc90da038a83bf0